test/factories/tasks.rb

```diff
@@ -1,9 +1,8 @@
 FactoryBot.define do
   factory :task do
-    author
-    assignee
     name { generate :string }
     description { generate :string }
-    expired_at { 1.month.from_now }
+    author_id { create :user }
```

Comment:
Strange, You have tried to initialize author_id with an instance of a user but not with his id.
You could define has_one associations in factory bot like this:
```
association :author, factory: :user
```